### PR TITLE
[392] [DEMO] Updated import statements

### PR DESCRIPTION
**Pull request by sagarlakshmipathy**
## What is the purpose of the pull request
- Fixes [392](https://github.com/apache/incubator-xtable/issues/392)

## Brief change log
- Fixed spark, delta, iceberg versions to follow the current pom.xml

## Verify this pull request
- Ran the notebook

### CODE REVIEW

File: demo/notebook/demo.ipynb [hunk 1]
```diff
@@ -10,27 +10,57 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```
Comment (the-other-tim-brown) [inline demo/notebook/demo.ipynb:13]: Can you remove these execution counts and outputs?

Comment (sagarlakshmipathy) [reply demo/notebook/demo.ipynb]: added a new notebook after clearing cells

**Comment by the-other-tim-brown** [comment]
@sagarlakshmipathy can you squash down to a single commit?

**Comment by daragu** [comment]
hi @the-other-tim-brown, you can choose `Squash and merge` option when merge.

图片

**Comment by the-other-tim-brown** [comment]
@daragu  that option was disabled due to some ASF processes.

**Pull request closed without merging**